src/components/ToolBoxComponents.js

```diff
@@ -4,64 +4,113 @@
 import React, { PropTypes } from 'react';
 import { Input, Button, ButtonToolbar } from 'react-bootstrap';
 
-export const CP_ButtonGroup = () => {
+/**
+ *
+ */
+export const CP_ButtonGroup = (props) => {
```

Comment:
I'm not sure where the "CP_*" convention came from, but it is generally a bad idea to use acronyms in variable names unless they are widely-adopted standards.